res/translations/source_copy_allow_list.tsv

```diff
@@ -0,0 +1,436 @@
+lang	source
```

Comment:
If you swap the columns, it's much more readable IMHO.

Reply:
Unfortunately not. This was my first version and I have swapped columns to have languages aligned.